scripts/backports.py

```diff
@@ -0,0 +1,112 @@
+# Generates Simple Release Notes for Kiali given a release version and Sprint Project.
```

Comment:
put this at the top so you can just run "backports.py" and have the operating system know to start it with python

```
#!/bin/python
```

Otherwise, this happens:

```
$ ./backports.py -h
./backports.py: line 12: import: command not found
./backports.py: line 13: import: command not found
./backports.py: line 14: import: command not found
./backports.py: line 16: syntax error near unexpected token `sys.argv'
./backports.py: line 16: `if len(sys.argv) != 5:'
```

Reply:
set the execute bit on the file too: `chmod +x backports.py`